DEVELOPMENT.md

```diff
@@ -141,22 +141,12 @@ Kubernetes cluster in your designated environment, if necessary.
 
 ### Deploy cert-manager
 
-1. Deploy `cert-manager` CRDs
```

Comment:
looks like the setup of our third_party scripts no longer has this file

Reply:
cc @ZhiminXiang

Reply:
this is because cert-manager does not provide the specific CRD file. And looks like it does not require to install CRD first https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests

Reply:
hm. Not a huge deal, but I think before it was split in two sections not just because it needed two steps, but also because some people wouldn't want to actually run the cert manager controllers for local dev? That's my reading of the previous docs, at least: "If you want to use the feature of automatically provisioning TLS for Knative services, you need to install the full cert-manager." (Also installing in one go [can apparently hang with some versions of kubectl](https://cert-manager.io/docs/installation/upgrading/upgrading-0.15-0.16/#issue-with-older-versions-of-kubectl), which might be worth a warning).

Upstream [does still ship](https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.crds.yaml) a crd-only file but I guess it's probably not that much extra resource just to run the controller in all cases so maybe this is better anyway 🤷.